rust_src/src/lread.rs

```diff
@@ -192,6 +234,15 @@ pub fn read(stream: LispObject) -> LispObject {
     }
 }
 
+/// Don't use this yourself
```

Comment:
```suggestion
/// Don't use this yourself.
```

The end punctuation was dropped.